ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```diff
@@ -11,10 +40,31 @@ tag_specification:
   name: "4.8"
   namespace: ocp
 tests:
-- as: unit
-  commands: ls /usr/local/kubebuilder/bin
+- as: verify
+  commands: |
+    make vendor && git diff --stat HEAD --ignore-submodules --exit-code
+  container:
+    from: src
+- as: unit-olm
+  commands: go env && make unit/olm
```

Comment:
Hmm, this appears to be problematic as we removed the nested vendor directory from VCS in the downstream repository so running any `go test ...` arguments attempt to build that tree as we go, but there are conflicts present in the modules defined.